docs/feature_pointing_device.md

```diff
@@ -89,15 +89,17 @@ POINTING_DEVICE_DRIVER = cirque_pinnacle_spi
-|`CIRQUE_PINNACLE_X_LOWER`        | (Optional) The minimum reachable X value on the sensor.                         | `127`                 |
-|`CIRQUE_PINNACLE_X_UPPER`        | (Optional) The maximum reachable X value on the sensor.                         | `1919`                |
-|`CIRQUE_PINNACLE_Y_LOWER`        | (Optional) The minimum reachable Y value on the sensor.                         | `63`                  |
-|`CIRQUE_PINNACLE_Y_UPPER`        | (Optional) The maximum reachable Y value on the sensor.                         | `1471`                |
-|`CIRQUE_PINNACLE_ATTENUATION`    | (Optional) Sets the attenuation of the sensor data.                             | `ADC_ATTENUATE_4X`    |
-|`CIRQUE_PINNACLE_TAPPING_TERM`   | (Optional) Length of time that a touch can be to be considered a tap.           | `TAPPING_TERM`/`200`  |
-|`CIRQUE_PINNACLE_TOUCH_DEBOUNCE` | (Optional) Length of time that a touch can be to be considered a tap.           | `TAPPING_TERM`/`200`  |
+| Setting                             | Description                                                                                      | Default                         |
+|-------------------------------------|--------------------------------------------------------------------------------------------------|---------------------------------|
+|`CIRQUE_PINNACLE_X_LOWER`            | (Optional) The minimum reachable X value on the sensor.                                          | `127`                           |
+|`CIRQUE_PINNACLE_X_UPPER`            | (Optional) The maximum reachable X value on the sensor.                                          | `1919`                          |
+|`CIRQUE_PINNACLE_Y_LOWER`            | (Optional) The minimum reachable Y value on the sensor.                                          | `63`                            |
+|`CIRQUE_PINNACLE_Y_UPPER`            | (Optional) The maximum reachable Y value on the sensor.                                          | `1471`                          |
+|`CIRQUE_PINNACLE_ATTENUATION`        | (Optional) Sets the attenuation of the sensor data.                                              | `ADC_ATTENUATE_4X`              |
+|`CIRQUE_PINNACLE_TAPPING_TERM`       | (Optional) Length of time that a touch can be to be considered a tap.                            | `TAPPING_TERM`/`200`            |
+|`CIRQUE_PINNACLE_TOUCH_DEBOUNCE`     | (Optional) Length of time that a touch can be to be considered a tap.                            | `TAPPING_TERM`/`200`            |
+|`CIRQUE_PINNACLE_RESET_ON_INIT_WAIT` | (Optional) Wait time in milliseconds after sending reset during init. Use 0 to skip.             | `30`                            |
+|`CIRQUE_PINNACLE_POSITION_MODE`      | (Optional) Absolute or Relative positioning mode. Use `CIRQUE_PINNACLE_{ABSOLUTE,RELATIVE}_MODE` | `CIRQUE_PINNACLE_ABSOLUTE_MODE` |
```

Comment:
Setting `CIRQUE_PINNACLE_POSITION_MODE` to `CIRQUE_PINNACLE_RELATIVE_MODE` will result in a compilation error. This option should therefore not be listed in the docs page.

Reply:
removed - thanks!